src/main/java/jsr223/cpython/entrypoint/EntryPoint.java

```diff
@@ -50,14 +50,14 @@ public static EntryPoint getInstance() {
         return ourInstance;
     }
 
-    private Map<String, Serializable> variabels = new HashMap<>();
+    private Map<String, Serializable> variables = new HashMap<>();
 
-    public Bindings bindings = new SimpleBindings();
+    private Bindings bindings = new SimpleBindings();
 
     //Add the objects to the gateway server
     //TODO not sure we still need this getVariables() because we already have the getBindings() whtich contains all
     public Map getVariables() {
-        return variabels;
+        return variables;
```

Comment:
Question to think about: Do you want to give access to the variables, or do you want to protect the variables and return a copy of the map?

Reply:
Hi, I want to give access to the variables here :)